app/components/FFXIVResults/ShortagePredictor/Results.tsx

```
@@ -0,0 +1,276 @@
+    },
+    {
+      columnId: 'quantity_decline_rate_per_hour',
+      header: 'Quantity Decline Rate per Hour'
+    },
+    {
+      columnId: 'current_quantity_vs_avg_percent',
+      header: 'Quantity Percent Available',
+      accessor: ({ getValue }) => {
+        const value = getValue()
+        if (typeof value === 'string') {
+          return <p>{parseFloat(value).toFixed(2)}%</p>
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)